### PR TITLE
Create script counting DISABLED tests amount

### PR DESCRIPTION
**Pull request by AByzhynar**
Created bash script which counts all DISABLED unit tests amount and shows it

Related issue: APPLINK-24966

### CODE REVIEW

**Comment by AByzhynar** [comment]
@OHerasym @LuxoftAKutsan Please review

File: tools/infrastructure/show_disabled.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+# Getting the list of all UT executables
+for test_file in $(ctest -N | awk '{print $3}'); do
+        full_path=`find . -name $test_file -type f 2>/dev/null`
+	if [ $full_path ] ; then
+        # Count amount of disabled tests cases
+		count_of_disabled=$(${full_path} --gtest_list_tests | grep DISABLED | wc -l)
+		if [ 0 != $count_of_disabled ] ; then
+		let "total_disabled = $total_disabled + $count_of_disabled"
+			echo $count_of_disabled " : " $test_file;
```
Comment (LuxoftAKutsan) [inline tools/infrastructure/show_disabled.sh]: Should we only calculate count of disabled tests or print their names to?

**Comment by LuxoftAKutsan** [comment]
Reviewed.

File: tools/infrastructure/show_disabled.sh (hunk 1 above)
Comment (AByzhynar) [inline tools/infrastructure/show_disabled.sh]: @LuxoftAKutsan Just calculate.  This script had such functionality in the past.  But it was deleted as redundant.

**Comment by OHerasym** [comment]
Reviewed.

**Comment by AByzhynar** [comment]
@AGritsevich @LevchenkoS @vlantonov @okozlovlux Please review

**Comment by AGritsevich** [comment]
Reviewed.

**Comment by LevchenkoS** [comment]
@AByzhynar Reviewed.

**Comment by okozlovlux** [comment]
@AByzhynar reviewed

**Comment by vlantonov** [comment]
Reviewed

**Pull request merged**